tests/e2e/tests/build/scripts-array.ts

```diff
@@ -48,5 +50,7 @@ export default function () {
       <script type="text/javascript" src="scripts.bundle.js"></script>
       <script type="text/javascript" src="vendor.bundle.js"></script>
       <script type="text/javascript" src="main.bundle.js"></script>
-    `));
+    `))
+     // ensure scripts aren't using script-loader when imported from the app
+    .then(() => expectFileToMatch('dist/main.bundle.js', 'console.log(\'string-script\');'));
```

Comment:
Yes, but in different ways:
```
// in scripts.bundle.js
/***/ 600:
/***/ function(module, exports) {
module.exports = "console.log('string-script');\r\n"

// in main.bundle.js
/***/ 614:
/***/ function(module, exports) {
console.log('scriptJsContent');
```

Script loader modifies the content such that it is a string, which makes it useless for normal imports.